### PR TITLE
Changed to upstream CentOS for now

### PR DESCRIPTION
**Pull request by BaptisteGerondeau**

### CODE REVIEW

File: files/machine_provision.yml [hunk 1]
```diff
@@ -89,6 +89,11 @@
 
                         if [ "${machine_type}" == "qdc" ]; then
                             machine_subarch=GrubWithOptionEfiboot
+                            if [ "${kernel_desc}" == "CentOS ERP 99" ]]; then
```
Comment (rengolin) [inline files/machine_provision.yml]: if ${os_type} == centos is better here.

File: files/preseeds/Centos.kickstart [hunk 2]
```diff
@@ -3,7 +3,8 @@
 
 # Use network installation
 url --url="http://mirror.centos.org/altarch/7/os/aarch64/"
-repo --name="ERP-RPM:staging" --baseurl=http://obs.linaro.org/ERP-RPM:/staging/staging/
+#repo --name="ERP-RPM:staging" --baseurl=http://obs.linaro.org/ERP-RPM:/staging/staging/
+repo --name="upstream" --baseurl=http://mirrors.coreix.net/centos-altarch/
```
Comment (rengolin) [inline files/preseeds/Centos.kickstart]: Why not use the standard AltArch? If that works for both ERP and non-ERP, we should just remove the ERP overlay (not comment out)

File: files/machine_provision.yml (hunk 1 above)
Comment (BaptisteGerondeau) [reply files/machine_provision.yml]: if os_type was a variable in this job, yes it'd be better.

File: files/preseeds/Centos.kickstart (hunk 2 above)
Comment (BaptisteGerondeau) [reply files/preseeds/Centos.kickstart]: Which one is the standard ?
removed

File: files/machine_provision.yml (hunk 1 above)
Comment (rengolin) [reply files/machine_provision.yml]: Then this is better changed in the OS job?

File: files/preseeds/Centos.kickstart (hunk 2 above)
Comment (rengolin) [reply files/preseeds/Centos.kickstart]: http://mirror.centos.org/altarch/7/os/aarch64/

**Comment by rengolin** [comment]
So, this change, what it needs is:
1. Pass ${os_type} down from os to machine job
2. Use ${os_type} to decide which CentOS to use
3. Select Upstream only for Amberwings, as D05's slaves are D03
4. Select ERP 119 for D0x, as that's the latest stable

**Pull request merged**